src/modules/coreHttpApi/openapi.yml

```diff
@@ -5805,6 +5944,27 @@ components:
       type: object
       description: The arbitrary JSON object which should be shared between creator of the Token and the recipient.
 
+    IdentityMetadata:
+      type: object
+      additionalProperties: false
+      properties:
+        reference:
+          allOf:
+            - $ref: "#/components/schemas/Address"
+          nullable: false
+          description: The address of the identity for that the metadata is stored for.
+        key:
+          type: string
+          nullable: true
+          description: An optional key to identify the metadata. Can be used to store multiple metadata entries for the same identity.
```

Comment:
Please make it a little clearer that there can be at most one IdentityMetadata object per identity key pair.

Reply:
how (and where) would you place that?

Reply:
For example, add the sentence "There can be at most one IdentityMetadata object per identity key pair." after your two sentences (which are great as description, btw!).